roles/backend/templates/backend.yml.j2

```diff
@@ -3,17 +3,9 @@
 # {{ managed }}
 #
 
-##
-# Sensu backend configuration
-##
-{# -------------------------------------- #}
-{% if backend_debug %}
-debug: {{ backend_debug }}
-{% endif %}
-{# -------------------------------------- #}
-{% if backend_log_level %}
-log-level: {{ backend_log_level }}
-{% endif %}
-{# -------------------------------------- #}
 state-dir: "/var/lib/sensu/sensu-backend"
```

Comment:
I know this line is not part of the current changeset, but I would rather see `/var/lib/sensu/sensu-backend` be the _default_ state directory and allow users to override it if they need to. We can probably solve this with a simple conditional:

```
{% if "state-dir" not in backend_config %}
state-dir: "/var/lib/sensu/sensu-backend"
{% endif %}
```

But feel free to solve this issue in a separate follow-up PR.

Reply:
👍 
Even though this puts us back to the if-land, I suppose it's not that bad to have just one-off instead of coming up with an elaborate way of baking into the vars the defaults.